dist/index.js

```diff
@@ -15172,6 +15174,9 @@ async function parseFile(file) {
                     (testcase.failure &&
                         testcase.failure._attributes &&
                         testcase.failure._attributes.message) ||
+                    (testcase.flakyFailure &&
+                        testcase.flakyFailure._attributes &&
+                        testcase.flakyFailure._attributes.message) ||
```

Comment:
@jmisur If you switch to Node 14 you can do: `testcase?.flakyFailure?._attributes?.message`   https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining